### PR TITLE
Adding Pill Box API

### PR DESCRIPTION
**Pull request by AnuradhaIyer**
API for adding and fetching Pill Details using UserName.

### CODE REVIEW

File: src/main/java/edu/sjsu/seekers/silversnug/service/PillBoxService.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import edu.sjsu.seekers.silversnug.response.PillBoxResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PillBoxService {
+
+
+    private final String SUCCESS = "SUCCESS";
```
Comment (nidhijmr) [inline src/main/java/edu/sjsu/seekers/silversnug/service/PillBoxService.java]: Can we define this in 1 single constant file?

File: src/main/java/edu/sjsu/seekers/silversnug/controller/PillBoxAPIController.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        @Autowired
+        PillBoxService pillBoxService;
+
+        @PostMapping("/SilverSnug/PillBox/addPill")
+        @ResponseBody
+        public GenericResponse addPill(@RequestBody PillBoxRequest request) {
+            return pillBoxService.savePill(request);
+
+               /*  pillBoxService.authenticate("Anu");
+                    return "Hello from Silver Snug!";*/
```
Comment (nidhijmr) [inline src/main/java/edu/sjsu/seekers/silversnug/controller/PillBoxAPIController.java]: please remove comments

File: src/main/java/edu/sjsu/seekers/silversnug/service/PillBoxService.java (hunk 1 above)
Comment (AnuradhaIyer) [reply src/main/java/edu/sjsu/seekers/silversnug/service/PillBoxService.java]: What do you suggest?

Comment (nidhijmr) [reply src/main/java/edu/sjsu/seekers/silversnug/service/PillBoxService.java]: define here: https://github.com/nidhijmr/SilverSnug/blob/master/src/main/java/edu/sjsu/seekers/silversnug/util/Constants.java

Comment (AnuradhaIyer) [reply src/main/java/edu/sjsu/seekers/silversnug/service/PillBoxService.java]: Done

Comment (AnuradhaIyer) [reply src/main/java/edu/sjsu/seekers/silversnug/service/PillBoxService.java]: @nidhijmr Please merge this so I can take latest changes

**Comment by nidhijmr** [comment]
@AnuradhaIyer : done

**Pull request merged**